ha/k8s_setup/ha_setup.py

```diff
@@ -306,6 +306,24 @@ def process(self):
         """
         sys.stdout.write('HA initialization is done.\n')
 
+class UpgradeCmd(Cmd):
```

Comment:
Can you please add "upgrade" in the usage() function?

Reply:
done